src/app/community/create/create.component.html

```diff
@@ -0,0 +1,27 @@
+<mat-card class="communityCreationForm">
+  <mat-card-title>Community Creation Form</mat-card-title>
+
+  <mat-card-content>
+    <mat-form-field id="communityNameField">
+      <input matInput placeholder="Community Name" [formControl]="nameFormControl" [(ngModel)]="nameText" maxlength="25" >
+      <mat-error *ngIf="nameFormControl.hasError('required')">
+        Name is <strong>required</strong>
+      </mat-error>
+    </mat-form-field>
+
+    <mat-form-field id="communityTitleField">
+      <input matInput placeholder="Community Title" [formControl]="titleFormControl" [(ngModel)]="titleText" maxlength="100">
+      <mat-error *ngIf="titleFormControl.hasError('required')">
+        Community Title is <strong>required</strong>
+      </mat-error>
+    </mat-form-field>
+
+    <mat-form-field id="comDescriptionField">
+      <textarea matInput placeholder="Community Description" [(ngModel)] = "descriptionText"></textarea>
+    </mat-form-field>
+
+  </mat-card-content>
+
+  <button mat-raised-button id="submitButton" color="primary" (click)="onCreateCommunity()"
```

Comment:
If you put this inside of a `mat-card-actions` here, you don't need to do any special styling on the button.

Reply:
mm, I tried using the mat-card-actions. It doesn't look as nice to me and I have to style it anyways. If you don't mind, I would like to just keep it like this since it's already done.